src/app/@ansyn/ol/mapSourceProviders/open-layers.map-source-provider.ts

```diff
@@ -52,7 +52,6 @@ export abstract class OpenLayersMapSourceProvider<CONF = any> extends BaseMapSou
 		});
 		const imageLayer = this.getImageLayer(source, extent);
 		this.removeExtraData(imageLayer);
-		tileLayer.set(IMAGE_PROCESS_ATTRIBUTE, imageLayer);
```

Comment:
Was this line removed on purpose?

Reply:
yes. i think the two line above also need to be removed.

Reply:
Can u remove these lines then?